### ISSUE TITLE
Console keeps dying

### ISSUE THREAD

**Issue by OverLordGoldDragon** [spyder-ide/spyder#13192]
Haven't seen something like this before; three consoles x'd out already, same error message. I still have the Spyder instance open, started another one to work with; if you have debug-info collection steps I could give it a go. 

```
Traceback (most recent call last):
File "D:\Anaconda\envs\tf2_env\lib\runpy.py", line 193, in _run_module_as_main
"__main__", mod_spec)
File "D:\Anaconda\envs\tf2_env\lib\runpy.py", line 85, in _run_code
exec(code, run_globals)
File "D:\Anaconda\envs\tf2_env\lib\site‑packages\spyder_kernels\console\__main__.py", line 11, in 
start.main()
File "D:\Anaconda\envs\tf2_env\lib\site‑packages\spyder_kernels\console\start.py", line 289, in main
import_spydercustomize()
File "D:\Anaconda\envs\tf2_env\lib\site‑packages\spyder_kernels\console\start.py", line 39, in import_spydercustomize
import spydercustomize
File "D:\Anaconda\envs\tf2_env\lib\site‑packages\spyder_kernels\customize\spydercustomize.py", line 22, in 
import logging
File "C:\Desktop\School\Deep Learning\DL_code\dev_tg\deeptrain\util\logging.py", line 10, in 
from .algorithms import deeplen, builtin_or_npscalar
ImportError: attempted relative import with no known parent package
```

![image](https://user-images.githubusercontent.com/16495490/86519313-a39c2c80-be4a-11ea-8670-361de60b339a.png)

**Comment by jitseniesen** [comment]
Looking at the last entry in the traceback, it looks like you have a file `logging.py` which shadows the one in the standard library.

**Comment by OverLordGoldDragon** [comment]
@jitseniesen I've had this file for months, and a new Spyder instance works just fine - so perhaps the "console starter" got "stuck" somewhere it shouldn't have, which is a bug. I'll also note this happened after a kernel restart was taking too long (20 secs) to show the `In [1]:`, so I just closed the console (which usually works just fine).

**Comment by ccordoba12** [comment]
Please rename `logging.py` to something else and try again.

**Comment by OverLordGoldDragon** [comment]
@ccordoba12 I'll try next time it happens again, but had to restart PC so the offending session's gone. So long as it's as rare as the decoding popup, not a bother.

**Comment by impact27** [comment]
Try setting the working directory to `C:\Desktop\School\Deep Learning\DL_code\dev_tg\deeptrain\util\` and then launch a new console. Does that replicate the issue?

**Comment by OverLordGoldDragon** [comment]
@impact27 RIP kernel - indeed.

Renaming to `_logging.py` enables successful console restart, @ccordoba12. Unsure it's applicable, but I've [tweaked](https://stackoverflow.com/a/62456280/10133797) default behavior with [`sys.meta_path`](https://docs.python.org/3/library/sys.html#sys.meta_path) - maybe Spyder could do something similar to avoid `cwd` conflicts with standard library.

**Comment by impact27** [comment]
I think ipython does something like that, so I am not sure why you have this problem. For reference: https://github.com/ipython/ipykernel/pull/209

**Comment by ccordoba12** [comment]
I fixed this problem in https://github.com/spyder-ide/spyder-kernels/pull/222. The fix will be available in 4.1.4.
